extensions/contentful/src/search.tsx

```diff
@@ -0,0 +1,284 @@
+      }
+    >
+      {entries?.items.map((entry) => {
+        const titleField = entry.fields[contentTypesDict[entry.sys.contentType.sys.id].displayField];
+        const title = titleField ? titleField["en-US"] : "title";
+        return (
+          <ContentfulContentEntryItem
+            key={entry.sys.id}
+            entry={entry}
+            title={title}
+            type={contentTypesDict[entry.sys.contentType.sys.id].name}
+            space={space}
+            showDetail={showDetail}
+            toggleDetail={toggleDetail}
+          />
+        );
+      })}
+    </List>
+  );
+}
+
+export interface ContentfulContentEntryItemProps {
+  space: string;
+  entry: ContentfulContentEntry;
+  title: string;
+  type: string;
+  toggleDetail: () => void;
+  showDetail?: boolean;
+}
+
+export const ContentfulContentEntryItem: FC<ContentfulContentEntryItemProps> = ({
+  entry,
+  title,
+  type,
+  space,
+  showDetail,
+  toggleDetail,
+}) => {
+  const entryUrl = `https://app.contentful.com/spaces/${space}/entries/${entry.sys.id}`;
+  const status = getEntryStatus(entry);
+  const statusColor = ColorMapping[status];
+
+  return (
+    <List.Item
+      icon={Icon.Document}
+      key={entry.sys.id}
+      title={title}
+      actions={
+        <ActionPanel>
+          <Action.OpenInBrowser url={entryUrl} />
+          <Action title={showDetail ? "Hide Detail" : "Show Detail"} onAction={toggleDetail} />
+        </ActionPanel>
+      }
+      accessories={
+        !showDetail
+          ? [
+              {
+                text: type,
+              },
+              {
+                tag: {
+                  value: status,
+                  color: statusColor,
+                },
+              },
+            ]
+          : []
+      }
+      detail={showDetail && <ContentfulContentEntryDetail entry={entry} status={status} type={type} />}
+    />
+  );
+};
+
+export interface ContentfulContentEntryDetailProps {
+  entry: ContentfulContentEntry;
+  status: ContentfulEntryStatus;
+  type: string;
+}
+
+export const ContentfulContentEntryDetail: FC<ContentfulContentEntryDetailProps> = ({ entry, status, type }) => {
+  const statusColor = ColorMapping[status];
+
+  return (
+    <List.Item.Detail
+      metadata={
+        <List.Item.Detail.Metadata>
+          <List.Item.Detail.Metadata.Label
+            title="id"
+            text={{
+              value: entry.sys.id,
+              color: Color.SecondaryText,
+            }}
+          />
+          <List.Item.Detail.Metadata.TagList title="type">
+            <List.Item.Detail.Metadata.TagList.Item text={type} />
+          </List.Item.Detail.Metadata.TagList>
+          <List.Item.Detail.Metadata.TagList title="status">
+            <List.Item.Detail.Metadata.TagList.Item text={status} color={statusColor} />
+          </List.Item.Detail.Metadata.TagList>
+          <List.Item.Detail.Metadata.Separator />
+
+          <List.Item.Detail.Metadata.Label title="Fields:" />
```

Comment:
```suggestion
          <List.Item.Detail.Metadata.Label title="Fields" />
```
To keep the consistence